ada/testsuite/testsuite_support/name_resolution_driver.py

```diff
@@ -13,7 +13,8 @@ class NameResolutionDriver(BaseDriver):
 
     @property
     def run_python(self):
-        return self.py_runner.is_python_api_available
+        return (self.py_runner.is_python_api_available and
```

Comment:
I'd rather go all the way and get rid of it at this stage ... It introduces a lot of complexity, the duplication of a lot of logic, all of that for a very dubious purpose. What do you think ?

Reply:
Agreed.